conf.d/fzf.fish

```diff
@@ -1,6 +1,6 @@
 # Set up the default, mnemonic key bindings unless the user has chosen to customize them
 if not set --query fzf_fish_custom_keybindings
-    set -l search_shell_variables '__fzf_search_shell_variables (set --show (set --names) | psub) (set --names)'
+    set -l search_shell_variables '__fzf_search_shell_variables (set --show | psub) (set --names)'
```

Comment:
Hmm, I'm not sure about this change. This means we need to dispaly a error message, presumably just `$history`.

Reply:
Yeah I'll just filter out history.

Don't know if you saw my commit comment but here it is
> don't try to show history variable because it adds complexity and because this plugin has a function dedicated to showing history already anyway